### PR TITLE
12.0 some improvements

### PR DESCRIPTION
**Pull request by Kev-Roche**
add changes from migration to 14 #994

### CODE REVIEW

**Comment by Kev-Roche** [comment]
cc @hparfr

File: shopinvader_product_attribute_set/models/product_filter.py [hunk 1]
```diff
@@ -10,7 +10,8 @@ class ProductFilter(models.Model):
     _inherit = "product.filter"
 
     based_on = fields.Selection(
-        selection_add=[("product_attribute", "Product Attribute")]
+        selection_add=[("product_attribute", "Product Attribute")],
+        ondelete={"product_attribute": "cascade"},
```
Comment (hparfr) [inline shopinvader_product_attribute_set/models/product_filter.py]: is v12 compatible ?

Comment (Kev-Roche) [reply shopinvader_product_attribute_set/models/product_filter.py]: good question, it is mandatory since V14, I don't know before, I'll check it.

Comment (Kev-Roche) [reply shopinvader_product_attribute_set/models/product_filter.py]: I deleted it

**Pull request merged**